tasks.py

```diff
@@ -0,0 +1,81 @@
+import os
+
+from robocorp import log
+from robocorp.tasks import task
+
+from PyPDFLibrary import PyPDFLibrary
+from PDFMinerLibrary import find_row, find_column
+
+PDF_INVOICE_FILE_PATH = os.path.join("devdata", "text-invoice.pdf")
+PDF_INVOICE_TABLE_FILE_PATH = os.path.join("devdata", "text-invoice-table.pdf")
```

Comment:
```suggestion
DEVDATA = Path("devdata")
PDF_INVOICE_FILE_PATH = DEVDATA / "text-invoice.pdf"
PDF_INVOICE_TABLE_FILE_PATH = DEVDATA / "text-invoice-table.pdf"
```
if you find it more valuable.

Reply:
Does this work in Windows? I usually rather just let the os.path.join construct the whole path.

Reply:
Yes it works, as that's a `Path` object now (not a string), which internally solves the `/` binary operator by calling a specific dunder method which will solve the path separators correctly.

Just keep in mind that this now will be a `Path` object and if you want its string form, you have to apply `str()` over the object.